chainstate/test-suite/src/tests/syncing_tests.rs

```diff
@@ -490,3 +490,52 @@ fn initial_block_download(#[case] seed: Seed) {
         assert!(!tf.chainstate.is_initial_block_download().unwrap());
     });
 }
+
+#[rstest]
+#[trace]
+#[case(Seed::from_entropy())]
+fn header_check_for_orphan(#[case] seed: Seed) {
+    utils::concurrency::model(move || {
+        let mut rng = make_seedable_rng(seed);
+        let mut tf = TestFramework::builder(&mut rng)
+            .with_chainstate_config(ChainstateConfig {
```

Comment:
Should we shorten this to `ChainstateConfig::default()`?

Reply:
@TheQuantumPhysicist asked to explicitly list all the fields to see all the places where adding a new field might have an effect.